parm/postcntrl_gfs_f00.xml

```diff
@@ -907,6 +907,36 @@
        <scale>3.0</scale>
     </param>
 
+    <param>
```

Comment:
@ericaligo-NOAA Do you think these six winter diag. variables will be calculated in model at F00? If not, you don't need adding them in the control files with f00.